config/default.yml

```diff
@@ -743,7 +743,11 @@ Naming/UncommunicativeMethodArgName:
   MinNameLength: 3
   AllowNamesEndingInNumbers: true
   # Whitelisted names that will not register an offense
-  AllowedNames: []
+  AllowedNames:
+    - io
+    - to
```

Comment:
I know I myself suggested `to`, but I think I’d prefer to keep only `io` and `id` by default — `to` is not a concept in Ruby, while `io` and `id` are.

Reply:
Sounds good. I'll remove the `to`.